Makefile

```
@@ -1,13 +1,24 @@
 GOPATH ?= $(HOME)/go
 GOBIN ?= $(GOPATH)/bin
 GOCMD := go
-GOBUILD := $(GOCMD) build
+
```

It works almost flawless (parsing date does not work) if you add:
```suggestion
ifeq ($(OS),Windows_NT)
SHELL := git-bash.exe
endif
```